techniques/systemSettings/misc/clockConfiguration/3.0/clockConfiguration.st

```diff
@@ -30,6 +30,11 @@ bundle agent check_clock_configuration
   vars:
       "ntpServers" slist => {&CLOCK_NTPSERVERS: { "&it&"};separator=", "&};
 
+      "target_timezone" string => ifelse( strcmp("&CLOCK_TIMEZONE&", "custom"), "&CLOCK_TIMEZONE_CUSTOM&", "&CLOCK_TIMEZONE&" );
```

Comment:
here we find out if we have chosen to use the "Custom" input field or we should go with the actual value of the Drop-Down menu.